mne/utils/check.py

```diff
@@ -300,13 +300,15 @@ def _check_preload(inst, msg):
         pass
     else:
         name = "epochs" if isinstance(inst, BaseEpochs) else "raw"
-        if not inst.preload:
+        if not getattr(inst, "preload", False):
```

Comment:
Why did you need to add this one?

Reply:
There was a test with RawArray where this caused an error

Reply:
I'll revert so I can see the failure... it seems odd and might indicate another bug

Reply:
@marsipu no failures in the test reverting this change, so maybe it wasn't necessary after all? Feel free to add a test case that showcases the issue otherwise

Reply:
Oh okay, can't reproduce the problem either now.